core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -320,6 +320,9 @@ final case class WriterT[F[_], L, V](run: F[(L, V)]) {
     G.map(
       F.traverse(run)(lv => G.tupleLeft(f(lv._2), lv._1))
     )(WriterT.apply)
+
+  def compare(that: WriterT[F, L, V])(implicit Ord: Order[F[(L, V)]]): Int =
+    Ord.compare(run, that.run)
```

Comment:
Is this a normal thing to add directly to the type? I actually don't know what our idiom is here.